model/patron.py

```diff
@@ -216,25 +213,6 @@ def works_on_loan_or_on_hold(self):
         loans = self.works_on_loan()
         return set(holds + loans)
 
-    def can_borrow(self, work, policy):
```

Comment:
This is the only place where the lending policy was applied. Logic around whether or not a patron can borrow a book now lives in circulation, although the main component piece of that logic -- whether a work is age-appropriate for a patron -- are now in Work.